### PR TITLE
[vertical-embed] fix: dynamically change base url

### PR DESCRIPTION
**Pull request by danarish4**
Following [this](https://github.com/wix-private/wix-ricos/pull/2090) PR

### CODE REVIEW

File: packages/plugin-vertical-embed/web/src/convertDuration.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { isNumber } from 'lodash';
```
Comment (danarish4) [inline packages/plugin-vertical-embed/web/src/convertDuration.ts:1]: exported from utils file for organization

File: packages/plugin-vertical-embed/web/src/verticalsDataBuilder.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+const contentTypeMapGetter = {
+  [verticalEmbedProviders.booking]: getBookingContent,
+  [verticalEmbedProviders.event]: getEventContent,
+  [verticalEmbedProviders.product]: getProductContent,
+};
+
+export const verticalsDataBuilder = (type, data, t, changeBaseUrl) => {
+  const { imageSrc, pageUrl, imageWidth, imageHeight } = data;
+  return {
+    url: changeBaseUrl ? changeBaseUrl(pageUrl) : pageUrl,
```
Comment (oded-soffrin) [inline packages/plugin-vertical-embed/web/src/verticalsDataBuilder.ts:30]: alternatively - 
```suggestion
export const verticalsDataBuilder = (type, data, t, changeBaseUrl = () => {}) => {
  const { imageSrc, pageUrl, imageWidth, imageHeight } = data;
  return {
    url: changeBaseUrl(pageUrl),
```

also. - how do you know whether this code is working? a unit-test on verticalDataBuilder will be good here

**Review by oded-soffrin — CHANGES_REQUESTED** [review]
comments in the PR
also - there must be some TS we need to change when adding more props to vertical embed config, right?

**Review by oded-soffrin — APPROVED** [review]
Pagaz

**Pull request merged**